lib/sqlalchemy/sql/functions.py

```diff
@@ -1072,6 +1077,19 @@ class GeoBuffer(GenericFunction):
         >>> print(func.geo.buffer())
         "ST_Buffer"()
 
+    Type parameters for this class as a
+    `generic type <https://peps.python.org/pep-0484/#generics>`_ can be passed
+    and should match the type seen in a :class:`_engine.Result`. For example::
+
+        class as_utc(GenericFunction[datetime.datetime]):
+            type = DateTime()
+            inherit_cache = True
+
+    The above indicates that the following expression returns a ``datetime``
+    object::
+
+        select(func.as_utc())
```

Comment:
we can probably use `conn.scalar(select(func.as_utc()))` since as is that returns a select

Reply:
Ah, my mistake, good catch; thanks!

I looked to see if `conn` was used elsewhere and I saw use of `connection` in another example in this file, so I wrote it as `connection.scalar(...)` to be consistent within the same file.